TLCGen.PluggedInItems/TLCGen.Generators.CCOL/CodeGeneration/CCOLGenerator.cs

```diff
@@ -195,7 +195,7 @@ public string GenerateSourceFiles(ControllerModel c, string sourcefilepath)
                         File.WriteAllText(Path.Combine(sourcefilepath, $"{c.Data.Naam}rgv.c"), GenerateRgvC(c), Encoding.Default);
                         _allFiles.Add($"{c.Data.Naam}rgv.c");
                     }
-                    if (c.PTPData.PTPKoppelingen?.Count > 0)
+                    if ((c.PTPData.PTPKoppelingen?.Count > 0) && c.PTPData.PTPKoppelingen.Any(x => x.Dummy != true))
```

Comment:
hiervan zou ik maken:
`!c.PTPData.PTPKoppelingen.Any(x => x.Dummy)`
dat is in lijn de stijl van de verdere code en is net iets compacter